### PR TITLE
React files setup

### PR DESCRIPTION
**Pull request by jaeroong**
added dependencies: react and webpack
added index.js
added App.jsx
added webpack.config
added index.html
edited App.jsx
added MainContainer.jsx
added test file enzyme.js
added jest and enzyme packages and nodemon

### CODE REVIEW

**Review by bryanfong-dev-old — APPROVED** [review]
looks good david!

File: package.json [hunk 1]
```diff
@@ -23,9 +24,27 @@
   },
   "homepage": "https://github.com/cipherchase/cipherchase#readme",
   "dependencies": {
+    "enzyme": "^3.10.0",
+    "jest": "^24.8.0",
+    "nodemon": "^1.19.1",
```
Comment (Donaldblanc) [inline package.json]: Hi Dave, lets move nodemon to a Dev dependencies.

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import MainContainer from './containers/MainContainer';
+
+class App extends Component {
```
Comment (Donaldblanc) [inline src/App.jsx]: Hi Dave, 

Can we refactor this into a functional component? This is ok but the legacy way to write react. 

import styled from 'styled-components';
import React from 'react';
const App = () => (
  <div>
  </div>
);
export default App;

**Review by Donaldblanc — CHANGES_REQUESTED** [review]
Hi Dave, this looks great, there area few changes that we need to make but this is a great start!

**Review by Donaldblanc — APPROVED** [review]
this looks great man thanks for making those changes 👍

**Pull request merged**